escalation-eda/escalation-swf/README.md

```diff
@@ -11,7 +11,21 @@ Email service is using [MailTrap Send email API](https://api-docs.mailtrap.io/do
 * Access to a Jira server (URL, user and [API token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/))
 * [in-cluster deployment only] Access to an OpenShift cluster with `admin` Role
 * An account to [MailTrap](https://mailtrap.io/home) with a [testing Inbox](https://mailtrap.io/inboxes) and an [API token](https://mailtrap.io/api-tokens)
-* Available or Running [Backstage Notification Service](https://github.com/mareklibra/janus-idp-backstage-plugins/commits/flpath560)
+* Available or Running [Janus IDP Backstage Notification Service](https://github.com/janus-idp/backstage-plugins/tree/4d4cb781ca9fc331a2c621583e9203f9e4585ee7), please note the following:
+  * This readme is made with Janus IDP version merged after Notification plugin's [PR #933](https://github.com/janus-idp/backstage-plugins/pull/933)
```

Comment:
The PR is already merged

Reply:
Thanks @pkliczewski. 

As there are tons of PRs approved/absorbed in upstream Janus IDP project, we have specifically noted the Janus IDP version immediately after Notifications Plugin PR was merged.

Hope it clarifies.